### DISCUSSION TITLE
📢 Version 5.3.1 is now online

### DISCUSSION THREAD

**Discussion by rockfordlhotka** [MarimerLLC/csla#1788] [General]
I just pushed a release for version 5.3.1 to NuGet.

This release includes four bug fixes reported by the community (thank you!).

If you reported or encountered these bugs, please test and let me know if you still encounter issues.

@ajj7060, @shushil, @ajohnstone-ks

**Comment by ajohnstone-ks** [comment]
@rockfordlhotka I updated to the 5.3.1 release and reverted the code changes I had implemented to work around these issues, and they all appear to be resolved.  Thanks for getting those out so quickly!
